### PR TITLE
LOG-2936: Add 5.6 logging CI

### PR DESCRIPTION
**Pull request by jcantrill**
This PR add CI for logging 5.6: https://issues.redhat.com/browse/LOG-2936

### CODE REVIEW

**Comment by jcantrill** [comment]
cc @Red-GV @periklis

File: ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-release-5.5.yaml [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    with: pipeline:logging-curator5
+promotion:
+  name: "5.5"
+  namespace: logging
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.11"
```
Comment (Red-GV) [inline ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-release-5.5.yaml:103]:
```suggestion
  latest:
    release:
      channel: stable
      version: "4.11"
```

**Review by Red-GV — COMMENTED** [review]
Just a small change and I think we're good.

File: ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-release-5.5.yaml (hunk 1 above)
Comment (jcantrill) [reply ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-release-5.5.yaml]: updated given i'm still resolving test failures

**Review by alanconway — APPROVED** [review]
/lgtm

**Pull request merged**